roles/shiftstack/defaults/main.yml

```diff
@@ -31,6 +31,7 @@ cifmw_shiftstack_manifests_dir: "{{ cifmw_shiftstack_basedir }}/manifests"
 cifmw_shiftstack_project_name: "shiftstack"
 cifmw_shiftstack_qa_gerrithub_change: ""
 cifmw_shiftstack_qa_repo: "https://review.gerrithub.io/shiftstack/shiftstack-qa"
+cifmw_shiftstack_rhoso_kubeconfig : "{{ cifmw_shiftstack_shiftstackclient_installation_dir }}/rhoso_kubeconfig"
```

Comment:
Please set the order so the `cifmw_shiftstack_shiftstackclient_installation_dir` is defined before use.